litmus-portal/frontend/src/pages/ApplicationDashboard/index.tsx

```diff
@@ -156,219 +280,601 @@ const DashboardPage: React.FC = () => {
+            </div>
+          </div>
+        ) : (
+          <div className={classes.root}>
+            <div className={classes.button}>
+              <BackButton />
+            </div>
 
-            <TopNavButtons
-              isInfoToggledState={isInfoOpen}
-              switchIsInfoToggled={(toggleState: Boolean) => {
-                setIsInfoOpen(toggleState);
+            <div className={classes.controlsDiv}>
+              <Typography variant="h4" style={{ fontWeight: 500 }}>
+                {`${selectedDashboardInformation.agentName} / `}
+                <Typography
+                  variant="h4"
+                  display="inline"
+                  style={{ fontStyle: 'italic' }}
+                >
+                  {selectedDashboardInformation.name}
+                </Typography>
+                <IconButton
+                  aria-label="more"
+                  aria-controls="long-menu"
+                  aria-haspopup="true"
+                  onClick={handleClick}
+                  data-cy="browseDashboardListOptions"
+                  className={classes.iconButton}
+                >
+                  <KeyboardArrowDownIcon
+                    className={classes.dashboardSwitchIcon}
+                  />
+                </IconButton>
+                <Menu
+                  id="long-menu"
+                  anchorEl={anchorEl}
+                  keepMounted
+                  open={open}
+                  onClose={handleClose}
+                  anchorOrigin={{
+                    vertical: 'bottom',
+                    horizontal: 'left',
+                  }}
+                  transformOrigin={{
+                    vertical: 'top',
+                    horizontal: 'right',
+                  }}
+                  getContentAnchorEl={null}
+                  classes={{ paper: classes.menuList }}
+                >
+                  {selectedDashboardInformation.dashboardListForAgent.map(
+                    (data: ListDashboardResponse) => {
+                      return (
+                        <MenuItem
+                          key={`${data.db_id}-monitoringDashboard`}
+                          value={data.db_id}
+                          selected={
+                            data.db_id === selectedDashboardInformation.id
+                          }
+                          onClick={() => {
+                            dashboard.selectDashboard({
+                              selectedDashboardID: data.db_id,
+                            });
+                            setSelectedDashboardInformation({
+                              ...selectedDashboardInformation,
+                              id: data.db_id,
+                            });
+                            setAnchorEl(null);
+                          }}
+                          className={classes.menuItem}
+                        >
+                          <div style={{ display: 'flex' }}>
```

Comment:
Avoid inline style

Reply:
same as above, class is meant for multiple properties or if we are reusing this style anywhere but thats not the case, this has been done to make the code base consistent, refer to workflow code base for similar usage @whitetiger1399